app/Resources/views/framework/ls_doc/_export_component.html.twig

```diff
@@ -2,8 +2,8 @@
     <p>Export Competency Framework “<b>{{ lsDoc.title }}</b>” as:</p>
     <div class="container-fluid" style="margin:15px 0"><div class="row">
         <div class="col-xs-3"><a href="{{ lsDoc.uri |local_uri }}.json" role="button" tabindex="1" class="btn btn-primary btn-block btn-export btn-export-case" style="white-space:normal; height:3.8em"  data-toggle="popover" data-trigger="hover" data-placement="bottom" data-html="true" data-content="Exports a JSON file using the IMS-standard format. This is the best format to use for archiving Frameworks.">Competency Framework Package (JSON)</a></div>
-        <div class="col-xs-3 hidden"><a href="javascript:alert('PDF export functionality coming soon...')" role="button" tabindex="2" class="btn btn-primary btn-block btn-export btn-export-pdf disabled" style="white-space:normal; height:3.8em"data-toggle="popover" data-trigger="hover" data-placement="bottom" data-html="true" data-content="Exports a styled PDF that can be printed, easily shared with others, and compared with other versions of the Framework.">Styled PDF</a></div><!-- {{ path('cfpackage_export2', {'id':lsDoc.id, '_format':'pdf'}) }} -->
         <div class="col-xs-3"><a href="{{ path('export_excel_file', {'id': lsDoc.id })}}" role="button" tabindex="3" class="btn btn-primary btn-block btn-export btn-export-excel" style="white-space:normal; height:3.8em"data-toggle="popover" data-trigger="hover" data-placement="bottom" data-html="true" data-content="Exports an Excel spreadsheet file that you can open and edit in spreadsheet programs such as Microsoft Excel.">Spreadsheet export</a></div><!-- {{ path('cfpackage_export2', {'id':lsDoc.id, '_format':'csv'}) }} -->
+        <div class="col-xs-3"><a onclick="SaltLocal.handlePdfFile('{{ lsDoc.uri |local_uri }}.json');" id="pdfgen" role="button" tabindex="3" class="btn btn-primary btn-block btn-export btn-export-pdf" style="white-space:normal; height:3.8em"data-toggle="popover" data-trigger="hover" data-placement="bottom" data-html="true" data-content="Exports a styled PDF that can be printed, easily shared with others, and compared with other versions of the Framework.">Styled PDF</a></div>
```

Comment:
Since this line was moved and had its tabindex changed to 3, the previous line should have its tabindex changed to 2.